Sources/Subprocess/Configuration.swift

```diff
@@ -509,17 +509,17 @@ internal enum StringOrRawBytes: Sendable, Hashable {
     func createRawBytes() -> UnsafeMutablePointer<CChar> {
         switch self {
         case .string(let string):
-#if os(Windows)
+            #if os(Windows)
```

Comment:
I'm sure we're very inconsistent about this one. I sort of like that `#if` is aligned differently - it calls out something that changes at a different time (compile time vs run time). Indenting them to me seems a little weird too, because we don't indent the lines under the 'else' like we would have if it were a traditional `if { }` block.

Reply:
Yeah fortunately the only settings are either
```
#if os(macOS)
var label = ""
#endif
```
or
```
#if os(macOS)
    var label = ""
#endif
```
I seem to see we use former more so I decided to go with that